wcfapp/WcfServiceWebApp/Web.config

```diff
@@ -0,0 +1,50 @@
+﻿<?xml version="1.0"?>
+<configuration>
+
+  <appSettings>
+    <add key="aspnet:UseTaskFriendlySynchronizationContext" value="true" />
+  </appSettings>
+  <system.web>
+    <compilation debug="true" targetFramework="4.6.2" />
+    <httpRuntime targetFramework="4.6.2"/>
+  </system.web>
+  <system.serviceModel>
+    <services>
+      <service name="WcfServiceWebApp.Service1">
+        <endpoint binding="basicHttpBinding" contract="WcfServiceWebApp.IService1" />
+        <endpoint binding="netTcpBinding" contract="WcfServiceWebApp.IService1" bindingConfiguration="noSecurityBind"/>
+        <endpoint address="mex" binding="mexTcpBinding" contract="IMetadataExchange" />
+      </service>
+    </services>
+    <bindings>
+      <netTcpBinding>
+        <binding name="noSecurityBind" portSharingEnabled="false">
+          <security mode="None" />
+        </binding>
```

Comment:
People might use this as a basis and end up with no security. This isn't a good example and we should call that out.

Reply:
Good point. I called it explicitly.